apps/meteor/client/views/marketplace/AppsList/AddonRequiredModal.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Modal } from '@rocket.chat/fuselage';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+import { useExternalLink } from '../../../hooks/useExternalLink';
+
+export type AddonActionType = 'install' | 'enable';
+
+type AddonRequiredModalProps = {
+	actionType: AddonActionType;
+	onDismiss: () => void;
+	onInstallAnyway: () => void;
+};
+
+const GET_ADONS_URL = 'https://go.rocket.chat/i/get-addons';
```

Comment:
Same here, move to https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/client/views/admin/subscription/utils/links.ts

Reply:
ok